website/docs/usage/rule-based-matching.md

```diff
@@ -232,14 +232,24 @@ following rich comparison attributes are available:
 >
 > # Matches tokens of length >= 10
 > pattern2 = [{"LENGTH": {">=": 10}}]
+>
+> # Match based on morph attributes
+> pattern3 = [{"MORPH": {"IS_SUBSET": ["Number=Singular", "Gender=Neuter"]}}]
+> doc = nlp(...)
+> # this will match because it's an exact match
+> doc[0].set_morph("Number=Singular|Gender=Neutral")
+> # this will not match
+> doc[0].set_morph("Number=Plural|Gender=Neutral")
+> # this will also not match because it's a superset
+> doc[0].set_morph("Number=Singular|Gender=Neutral|Politeness=Informal")
```

Comment:
I think it would be clearer (and also fix the inconsistency around `Neutral`/`Neuter`) to use the UD inventory for this example, and also to include a proper subset to demonstrate. I don't think the `doc` / `set_morph` bits fit into this particular sidebar, so I, uh, sort of rewrote the whole thing in the suggestion, but I am less concerned about that.

```suggestion
> pattern3 = [{"MORPH": {"IS_SUBSET": ["Number=Sing", "Gender=Neut"]}}]
> # "", "Number=Sing" and "Number=Sing|Gender=Neut" will match as subsets
> # "Number=Plur|Gender=Neut" will not match
> # "Number=Sing|Gender=Neut|Polite=Infm" will not match because it's a superset
```

Reply:
Thank you, that's great!